packages/vertica-nodejs/mochatest/integration/client/vertica-connection-params-tests.js

```diff
@@ -105,12 +105,17 @@ describe('vertica backup_server_node connection parameter', function() {
   })
 })
 
-describe('vertica client_os_hostname connection parameter', function() {
-  it('is automatically determined and sent in the startup packet', function() {
+describe('vertica-nodejs handling auditing connection properties', function() {
+  it('are provided automatically when establishing a connection', function() {
     const client = new vertica.Client()
     client.connect()
-    client.query("SELECT client_os_hostname FROM current_session", (err, res) => {
+    client.query("SELECT client_pid, client_type, client_version, client_os, client_os_user_name, client_os_hostname FROM CURRENT_SESSION", (err, res) => {
       if (err) assert(false)
+      assert.equal(res.rows[0].client_pid, process.pid)
+      assert.equal(res.rows[0].client_type, "Node.js Driver")
+      assert.equal(res.rows[0].client_version, vertica.version)
```

Comment:
@williammatherjones Now our vertica-nodejs package exports its current version as defined in package.json. This allows me to reference it programmatically. So this test is now comparing the value of client_version that the driver sends and is stored in the sessions table with the value in package.json. This means if we forgot to change the version in the startup request packet, the test would fail and couldn't merge the PR for updating the version until we fixed what is being sent to the server

Reply:
That's a great solution!   Thanks @DMickens